src/shared/components/query/CancerStudySelector.tsx

```diff
@@ -0,0 +1,267 @@
+import * as _ from "lodash";
+import * as React from "react";
+import * as ReactDOM from "react-dom";
+import Dictionary = _.Dictionary;
+import {TypeOfCancer, CancerStudy} from "../../api/CBioPortalAPI";
+import {ITreeDescriptor, default as DescriptorTreeNode} from "../tree/DescriptorTreeNode";
+import * as ReactBootstrap from 'react-bootstrap';
+import {BootstrapTable} from "react-bootstrap-table";

```

Comment:
ok.  admittedly the table component situation sucks.  we're going to have to use FB's fixed data table (in fact our enhancement of it) for tables where performance is a factor because of huge # of rows/columns.   it would be nice to keep this to a minimum.